readmes/TUNNELS.md

```
@@ -71,12 +71,37 @@ export const app = r
   .build();
 ```
 
-Client: call it from Node or the browser.
+Client: call it from Node or the browser using the factory (recommended) or direct client creation.
+
+**Recommended: Use HTTP Client Factory with DI**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)